### DISCUSSION TITLE
Is webpack offers project in GSOC 25

### DISCUSSION THREAD

**Discussion by Advaitgaur004** [webpack/webpack#18946] [Q&A]
Last year, Webpack had some decent ideas; three of them are unprocessed while the others, such as Webpack: Universal Targets and Webpack: Destructuring, remain untouched. Is there any chance this will happen in GSOC '25?

**Comment by alexander-akait** [comment]
`Destructuring` is solved, `Universal Targets` still make sense, but there are no many issues - wasm loanding and hmr

**Reply by Advaitgaur004** [reply]
So is there project ideas that needs to be cater

**Reply by alexander-akait** [reply]
Currently we don't have it, only - https://github.com/orgs/webpack/projects/4
